Packs/GLPI/Integrations/GLPI/GLPI.py

```diff
@@ -0,0 +1,973 @@
+                        file = client.download_document(ticket_doc.get('documents_id'), filename=document.get('filename'))
+                        demisto.debug('file fetched')
+                        filepath, filename = os.path.split(file)
+                        f = open(file, 'rb')
+                        data = f.read()
+                        entries.append(fileResult(filename, data))
+
+        comments_result = client.get_ticket_comments(ticket_id)
+        if comments_result:
+            for note in comments_result:
+                if 'Mirrored from Cortex XSOAR' not in note.get('content') and note.get('date_mod') > formated_date:
+                    comments_context = {'comments_and_work_notes': unescape(note.get('content'))}
+                    entries.append({
+                        'ContentsFormat': formats['html'],
+                        'Type': entryTypes['note'],
+                        'Contents': unescape(note.get('content')),
+                        'Note': True,
+                        'EntryContext': comments_context
+                    })
+        demisto.debug(f'Pull result is {new_incident_data}')
+        return GetRemoteDataResponse(new_incident_data, entries)
+    except Exception as e:
+        raise DemistoException('Error : ' + str(e))
+
+
+def update_remote_system_command(client: Client, args: Dict[str, Any], params: Dict[str, Any]) -> str:
+    """update-remote-system command: pushes local changes to the remote system
+
+    :type client: ``Client``
+    :param client: XSOAR client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['data']`` the data to send to the remote system
+        ``args['entries']`` the entries to send to the remote system
+        ``args['incidentChanged']`` boolean telling us if the local incident indeed changed or not
+        ``args['remoteId']`` the remote incident id
+
+    :return:
+        ``str`` containing the remote incident id - really important if the incident is newly created remotely
+
+    :rtype: ``str``
+    """
+    parsed_args = UpdateRemoteSystemArgs(args)
+    ticket_id = parsed_args.remote_incident_id
+    if parsed_args.delta:
+        demisto.debug(f'Got the following delta keys {str(list(parsed_args.delta.keys()))}')
+
+    demisto.debug(f'Sending incident with remote ID [{parsed_args.remote_incident_id}] to remote system\n')
+    new_incident_id: str = parsed_args.remote_incident_id
+    updated_incident = {}
+
+    if not parsed_args.remote_incident_id or parsed_args.incident_changed:
+        if parsed_args.remote_incident_id:
+            old_incident = client.get_ticket(parsed_args.remote_incident_id)
+            for changed_key in parsed_args.delta.keys():
+                old_incident[changed_key] = parsed_args.delta[changed_key]  # type: ignore
+            parsed_args.data = old_incident
```

Comment:
Notice that `parsed_args.data` holds everything that passed the mapper, and `parsed_args.delta` holds all the fields that were changed. `delta` can hold some server fields that have changed (for example, `closeReason` that appears when closing an incident in XSOAR and proving a reason for it). 

Do you want to take the values that are in `delta`, or the ones that are in `data`?
Maybe holding a list that has the names of the fields you want to take will be a good solution.
If `update_ticket` uses only the relevant fields then it is not a real problem.